### PR TITLE
add aggregated cards

### PR DESCRIPTION
**Pull request by jinyz8888**
The cards work but have different sizes. Tried to set them to be the same size, but didn't work, have no idea to optimize it.

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -59,28 +102,33 @@
...
-    dbc.Row([
+    """ dbc.Row([
         dbc.Col(card, md=12),
     ], style=ROW_STYLE),
     dbc.Row([
         dbc.Col(card, md=4),
         dbc.Col(card, md=4),
         dbc.Col(card, md=4),
-    ], style=ROW_STYLE),
+    ], style=ROW_STYLE), """
```
Comment (SoloSynth1) [inline src/app.py:131]: Please remove the unused code block instead of commenting it out

File: src/app.py [hunk 2]
```diff
@@ -59,28 +102,33 @@
...
 
-test_graph = dcc.Graph(id='example-graph', figure=fig)
+# test_graph = dcc.Graph(id='example-graph', figure=fig)
 
-card = dbc.Card(children=[
+""" card = dbc.Card(children=[
     html.B(children="Test Graph"),
     test_graph
-])
+]) """
```
Comment (SoloSynth1) [inline src/app.py:110]: `card` is still needed for placeholders.

File: src/app.py [hunk 3]
```diff
@@ -59,28 +102,33 @@
     style=SIDEBAR_STYLE,
 )
 
-test_graph = dcc.Graph(id='example-graph', figure=fig)
+# test_graph = dcc.Graph(id='example-graph', figure=fig)
```
Comment (SoloSynth1) [inline src/app.py:105]: Please remove the unused code instead of commenting it out
```suggestion
```

File: src/app.py [hunk 4]
```diff
@@ -1,20 +1,62 @@
...
 
-fig = px.bar(df, x="Fruit", y="Amount", color="City", barmode="group")
+def create_card(title, value, id_value):
+    return dbc.Card(
+        [
+            html.H4(title, className="card-title", style={"color": "black"}),
+            html.H2(value, className="card-value", id=id_value),
+        ],
+        body=True,
+        style={"textAlign": "center", "color": "red"}
```
Comment (SoloSynth1) [inline src/app.py:52]: It would be best if you can combine the styles in this dictionary with the `CARD_STYLE`.

File: src/app.py [hunk 5]
```diff
@@ -1,20 +1,62 @@
...
+    )
+
+# fig = px.bar(df, x="Fruit", y="Amount", color="City", barmode="group")
```
Comment (SoloSynth1) [inline src/app.py:55]: Please remove the unused code instead of commenting it out.
```suggestion
```

**Review by SoloSynth1 — CHANGES_REQUESTED** [review]
@jinyz8888 Please see comments. I can see you commented out the `card` variable but I think it is still referenced in the `app.layout`. Have you tried to run the app? Did it render successfully?

I can also see for now the numbers are hardcoded to calculate fold between 2001 and 2015. No changes are needed for now but we definitely need to change it later on when we implement the filters.

**Comment by SoloSynth1** [comment]
@jinyz8888 As for the difference in card sizes, I would say don't worry about this for now. Later on we can make it more consistent via CSS.

File: src/app.py (hunk 1 above)
Comment (SoloSynth1) [reply src/app.py]: Resolving this as the commented lines were removed in the latest commit.

File: src/app.py (hunk 2 above)
Comment (SoloSynth1) [reply src/app.py]: Resolving this as the commented lines were removed in the latest commit.

File: src/app.py (hunk 3 above)
Comment (SoloSynth1) [reply src/app.py]: Resolving this as the commented lines were removed in the latest commit.

File: src/app.py (hunk 5 above)
Comment (SoloSynth1) [reply src/app.py:55]: Resolving this as the commented lines were removed in the latest commit.

**Comment by SoloSynth1** [comment]
I have made the cards consistent in height, but it looks wack as the titles are too large. This will be fixed in another PR.

image

File: src/app.py (hunk 4 above)
Comment (SoloSynth1) [reply src/app.py]: extracted the styles and transferred them into an external CSS file. Resolving this.

**Review by SoloSynth1 — APPROVED** [review]
Let's merge it into `main` for now. The aggregated values are static right now so we have to create some callbacks during implementing the filters.

**Comment by SoloSynth1** [comment]
Merged into `main`. Thanks @jinyz8888 !

**Comment by jinyz8888** [comment]
Thanks @SoloSynth1 !

**Pull request merged**